File: IOS/TMStore/TMStore/Classes/Models/QuantityRule.m
//
//  QuantityRule.m
//  TMStore
//
//  Created by Vikas Patidar on 07/11/16.
//  Copyright © 2016 Twist Mobile. All rights reserved.
//

#import <Foundation/Foundation.h>
#import "QuantityRule.h"

@implementation QuantityRule
- (id)init {
    if (self = [super init]) {
        self.orderrideRule = NO;
        self.stepValue = 1;
        self.minQuantity = 0;
        self.maxQuantity = 0;
        self.minOutOfStock = 0;
        self.maxOutOfStock = 0;
    }
    return self;
}
@end

File: Android/_setup/_ProjectName_/src/main/java/com/twist/dataengine/entities/QuantityRule.java
package com.twist.dataengine.entities;

/**
 * Created by Twist Mobile on 06-Sep-16.
 */

public class QuantityRule {

    private boolean overrideRule = false;
    private int stepValue = 1;
    private int maxQuantity = 0;
    private int minQuantity = 0;
    private int minOutOfStock = 0;
    private int maxOutOfStock = 0;

    public int getStepValue() {
        return stepValue;
    }

    public void setStepValue(int stepValue) {
        this.stepValue = stepValue;
    }

    public int getMaxQuantity() {
        return maxQuantity;
    }

    public void setMaxQuantity(int maxQuantity) {
        this.maxQuantity = maxQuantity;
    }

    public int getMinQuantity() {
        return minQuantity;
    }

    public void setMinQuantity(int minQuantity) {
        this.minQuantity = minQuantity;
    }

    public int getMinOutOfStock() {
        return minOutOfStock;
    }

    public void setMinOutOfStock(int minOutOfStock) {
        this.minOutOfStock = minOutOfStock;
    }

    public int getMaxOutOfStock() {
        return maxOutOfStock;
    }

    public void setMaxOutOfStock(int maxOutOfStock) {
        this.maxOutOfStock = maxOutOfStock;
    }

    public boolean isOverrideRule() {
        return overrideRule;
    }

    public void setOverrideRule(boolean overrideRule) {
        this.overrideRule = overrideRule;
    }
}
